README.rst

```diff
@@ -0,0 +1,47 @@
+Python access network interfaces
+================================
+
+General information
+-------------------
+
+Pyiface is a package that exposes the network interfaces of the operating
+system in a easy to use and transparent way Typical usage looks like this:
+
+.. code-block:: python
+
+    #!/usr/bin/env python
+    
+    import pyiface
+    from pyiface.ifreqioctls import IFF_UP
```

Comment:
* `pyiface.ifreqioctls` it's not visible
* also socket should be imported

```python3
import socket
IFF_UP = 0x1
```
I think, we should make `pyiface.ifreqioctls` visible. I can do that.

Reply:
Actually, it works now :+1:

Reply:
@alexvasiu so release can be made now?

Reply:
Sure :) I'll make a release now in **pip** and then I'll also update recipe in `meta-openembedded`